mlflow/models/model.py

```diff
@@ -432,6 +434,7 @@ def saved_input_example_info(self, value: Dict[str, Any]):
     def model_size_bytes(self) -> Optional[int]:
         """
         An optional integer that represents the model size in bytes
+
```

Comment:
unrelated to the PR, but found a bug in the existing docs. without the newline, the getter/setter lines don't get rendered properly:

https://mlflow.org/docs/latest/python_api/mlflow.models.html#mlflow.models.Model.model_size_bytes